kategory/src/main/kotlin/kategory/free/Free.kt

```diff
@@ -85,3 +85,5 @@ fun <M, S, A> Free<S, A>.foldMap(f: FunctionK<S, M>, MM: Monad<M>): HK<M, A> =
 
 fun <S, A> A.free(): Free<S, A> =
         Free.pure<S, A>(this)
+
+fun <F, A> Free<F, A>.run(M: Monad<F>): HK<F, A> = this.foldMap(FunctionK.id(), M)
```

Comment:
I'd also provide a version for the HKs like the one in https://github.com/kategory/kategory/pull/157